src/HypothesisClient/Model/ModelTrait.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace eLife\HypothesisClient\Model;
+
+trait ModelTrait
+{
+    protected $new = false;
+
+    public function isNew(): bool
+    {
+        return $this->new;
+    }
+
+    public function setNew()
```

Comment:
Think a custom serializer to add it in the constructor rather than making it stateful would be a better approach.